python/Makefile

```diff
@@ -95,17 +95,23 @@ push_conda:
 	@echo "Alternatively use anaconda upload to publish on own channel"
 
 setup_linter:
-	pip install black==20.8b1
+	pip install black==20.8b1 isort==5.7.0
 
 fmt:
 	black \
 		./ ../testing ../operator/helm ../operator/hack \
-		--exclude "(testing/scripts/proto|seldon_core/proto/|.eggs|.tox)"
+		--exclude "(testing|scripts|proto|seldon_core/proto/|.eggs|.tox)"
```

Comment:
Should be `testing/scripts`  not `testing|scripts`

Reply:
sorry for confusion, fixed.

Reply:
Hmm... actually, maybe just `scripts` is enough because this applies when being inside `../testing` already.